docs/source/overview.md

```diff
@@ -0,0 +1,77 @@
+# SimPhoNy: interoperability powered by ontology
+
+SimPhoNy is an ontology-based framework aimed at enabling interoperability between different simulation and data management tools, with a focus on materials science.
+
+
+
+## What can SimPhoNy be used for?
+
+### Manipulate ontology-based linked data, a format well suited for [FAIR data principles](https://en.wikipedia.org/wiki/FAIR_data).
+
+
+[Linked data](https://en.wikipedia.org/wiki/Linked_data) is a format for structured data that facilitates the interoperability among different data sources. In particular, the data is structured as a directed graph, consistent of nodes and labeled arcs. With SimPhoNy, you can not only manipulate this linked data, **but also transform existing non-linked data into linked data**.
+
+To better understand the idea of linked data, just have a quick glance at the toy example below.
```

Comment:
I really enjoyed the graphic, you have. But I think it would be nice to elaborate on it with 1-2 sentences to catch the attention of the reader a bit more on it, rather than just pointing to it

Reply:
Check [ea4b26f](https://github.com/simphony/docs/pull/121/commits/ea4b26fcde68927b8eb2dbe197ca6d492ffe9b76) and [0d72324](https://github.com/simphony/docs/pull/121/commits/0d72324888d1420d0929fe59971c0b9e4e335a24). The transition from text to picture should be less abrupt now.